sandbox/templates/richie/base.html

```diff
@@ -12,12 +12,95 @@
     </head>
     <body>
         {% cms_toolbar %}
-        <div>
-            {% show_menu 0 100 100 100 %}
-            {% block content %}
+        {% block body_header %}
+        <nav class="topbar" id="main-menu">
+            <div class="topbar__brand">
+                <button class="topbar__hamburger"
+                        data-target="main-menu" aria-label="menu" aria-expanded="false">&#8801;</button>
+                <a href="/" class="topbar__brand__link">
+                    <img src="{% static "images/logo.png" %}" class="topbar__brand__logo" alt="">
```

Comment:
missing alt?

Reply:
my bad, commonly it's me that shouts on webdesigners which forget it..